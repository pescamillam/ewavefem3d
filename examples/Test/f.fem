# Number of nodes and number of elements
  nNod = 176   nEl = 2

# Plane stress state, 2D problem 
 stressState = PLSTRESS

# Enable thermal loading option
  thermalLoading = N

# Material properties: 
# material name, elasticity modulus, Poisson's ratio and
#    coefficient of thermal expansion
  material = 1   1   0.3   0.1  

# Nodal coordinates
  nodCoord = 
0 2 
0 1.8 
0 1.6000000000000001 
0 1.4000000000000001 
0 1.2000000000000002 
0 1.0000000000000002 
0 0.80000000000000016 
0 0.60000000000000009 
0 0.40000000000000002 
0 0.20000000000000001 
0 0 
0.20000000000000007 2 
0.19999999999999973 1.7999999999999998 
0.19999999999999929 1.6000000000000001 
0.19999999999999901 1.3999999999999999 
0.19999999999999854 1.2000000000000002 
0.19999999999999821 1.0000000000000002 
0.19999999999999785 0.80000000000000016 
0.19999999999999746 0.60000000000000009 
0.19999999999999701 0.39999999999999969 
0.19999999999999662 0.19999999999999973 
0.19999999999999629 0 
0.40000000000000013 2 
0.39999999999999974 1.7999999999999998 
0.39999999999999941 1.6000000000000001 
0.39999999999999908 1.3999999999999999 
0.39999999999999875 1.2 
0.39999999999999841 0.99999999999999978 
0.39999999999999802 0.79999999999999982 
0.39999999999999769 0.59999999999999964 
0.39999999999999736 0.39999999999999947 
0.39999999999999702 0.19999999999999954 
0.39999999999999658 0 
0.6000000000000002 2 
0.59999999999999987 1.7999999999999994 
0.59999999999999953 1.5999999999999992 
0.5999999999999992 1.3999999999999999 
0.59999999999999887 1.1999999999999997 
0.59999999999999842 1 
0.59999999999999809 0.79999999999999938 
0.59999999999999776 0.59999999999999953 
0.59999999999999742 0.39999999999999941 
0.5999999999999972 0.19999999999999923 
0.59999999999999687 0 
0.80000000000000016 2 
0.79999999999999982 1.7999999999999994 
0.79999999999999949 1.5999999999999992 
0.79999999999999916 1.3999999999999997 
0.79999999999999916 1.2 
0.79999999999999871 1 
0.79999999999999827 0.7999999999999996 
0.79999999999999816 0.5999999999999992 
0.7999999999999976 0.39999999999999925 
0.79999999999999738 0.19999999999999904 
0.79999999999999716 0 
1 2 
0.99999999999999956 1.7999999999999996 
0.99999999999999944 1.6000000000000001 
0.99999999999999911 1.3999999999999997 
0.99999999999999889 1.1999999999999993 
0.99999999999999889 0.99999999999999944 
0.99999999999999845 0.79999999999999916 
0.99999999999999811 0.59999999999999942 
0.99999999999999778 0.39999999999999891 
0.99999999999999756 0.19999999999999885 
0.99999999999999745 0 
1.2000000000000002 2 
1.3999999999999999 2 
1.6000000000000001 2 
1.7999999999999998 2 
2 2 
2.1999999999999997 2 
2.3999999999999999 2 
2.5999999999999996 2 
2.7999999999999998 2 
3 2 
1.2000000000000002 1.7999999999999996 
1.1999999999999997 1.5999999999999996 
1.1999999999999993 1.3999999999999997 
1.1999999999999991 1.1999999999999993 
1.1999999999999993 0.99999999999999944 
1.1999999999999984 0.79999999999999916 
1.1999999999999984 0.59999999999999876 
1.199999999999998 0.39999999999999875 
1.199999999999998 0.19999999999999857 
1.1999999999999977 0 
1.3999999999999997 1.7999999999999996 
1.5999999999999992 1.7999999999999996 
1.7999999999999996 1.7999999999999996 
2 1.7999999999999996 
2.1999999999999993 1.7999999999999996 
2.3999999999999999 1.7999999999999996 
2.5999999999999996 1.7999999999999998 
2.7999999999999998 1.7999999999999998 
3 1.7999999999999996 
1.3999999999999999 1.6000000000000001 
1.3999999999999995 1.3999999999999997 
1.399999999999999 1.1999999999999991 
1.399999999999999 0.99999999999999911 
1.3999999999999988 0.79999999999999882 
1.3999999999999986 0.59999999999999876 
1.3999999999999986 0.39999999999999858 
1.3999999999999986 0.19999999999999835 
1.3999999999999981 0 
1.5999999999999996 1.6000000000000001 
1.7999999999999996 1.5999999999999992 
1.9999999999999991 1.5999999999999992 
2.1999999999999993 1.5999999999999996 
2.3999999999999995 1.5999999999999992 
2.5999999999999996 1.5999999999999996 
2.7999999999999998 1.5999999999999996 
3 1.5999999999999992 
1.5999999999999996 1.3999999999999995 
1.5999999999999992 1.1999999999999993 
1.599999999999999 0.99999999999999889 
1.5999999999999988 0.79999999999999893 
1.599999999999999 0.59999999999999876 
1.5999999999999983 0.39999999999999836 
1.5999999999999988 0.19999999999999812 
1.5999999999999983 0 
1.7999999999999994 1.3999999999999997 
1.9999999999999996 1.3999999999999997 
2.1999999999999993 1.3999999999999995 
2.3999999999999995 1.3999999999999995 
2.5999999999999996 1.3999999999999992 
2.7999999999999994 1.3999999999999986 
3 1.3999999999999988 
1.7999999999999994 1.1999999999999991 
1.7999999999999994 0.99999999999999878 
1.7999999999999989 0.79999999999999882 
1.7999999999999994 0.59999999999999842 
1.7999999999999989 0.39999999999999808 
1.7999999999999989 0.19999999999999787 
1.7999999999999985 0 
1.9999999999999996 1.1999999999999991 
2.1999999999999993 1.1999999999999988 
2.3999999999999999 1.1999999999999988 
2.5999999999999996 1.1999999999999986 
2.7999999999999998 1.1999999999999982 
3 1.1999999999999984 
1.9999999999999996 0.99999999999999878 
1.9999999999999996 0.79999999999999827 
1.9999999999999996 0.59999999999999831 
1.9999999999999991 0.39999999999999791 
1.9999999999999989 0.19999999999999762 
1.9999999999999989 0 
2.1999999999999988 0.99999999999999845 
2.399999999999999 0.99999999999999845 
2.6000000000000001 0.99999999999999822 
2.7999999999999994 0.99999999999999811 
3 0.999999999999998 
2.1999999999999993 0.79999999999999849 
2.1999999999999993 0.59999999999999787 
2.1999999999999997 0.39999999999999775 
2.1999999999999993 0.19999999999999737 
2.1999999999999993 0 
2.3999999999999995 0.79999999999999805 
2.6000000000000001 0.79999999999999782 
2.7999999999999998 0.79999999999999805 
3 0.7999999999999976 
2.3999999999999999 0.59999999999999765 
2.3999999999999995 0.39999999999999741 
2.3999999999999986 0.19999999999999712 
2.3999999999999995 0 
2.6000000000000001 0.59999999999999765 
2.7999999999999989 0.59999999999999731 
3 0.5999999999999972 
2.6000000000000001 0.39999999999999725 
2.5999999999999996 0.19999999999999696 
2.5999999999999996 0 
2.7999999999999998 0.39999999999999708 
3 0.3999999999999968 
2.8000000000000003 0.19999999999999662 
2.7999999999999998 0 
3 0.1999999999999964 
3 0 

# Element data: element type, material, connectivities
  elCon = QUAD8 1  1  4  6  7  8  5  3  2
          QUAD8 1  6  9 11 12 13 10  8  7

# Constraints: direction, value, number of constraints, 
#    node numbers
  constrDispl = x 0.0 2  1 -3
  constrDispl = y 0.0 5  1 4 6 9 11
  
end  
  
# Load
  loadStep = 1
  
# Surface load: direction, element number, number of face
#    nodes, face node numbers, intensities  
  surForce = x  2 3 11 12 13  1 1 1

# Nodal temperatures
  
end

