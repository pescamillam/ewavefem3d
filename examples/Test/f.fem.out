Date: 2016-11-20  Time: 20:39:31

Number of elements    nEl = 2
Number of nodes      nNod = 176
Number of dimensions nDim = 2
Memory for global matrix:    0.01 MB
Loadstep 1

Solution time =       0.05 s
